src/rebar3_hex_error.erl

```diff
@@ -21,5 +21,5 @@ format_error({Cmd, missing_required_params}) ->
     io_lib:format("Required parameters for the ~ts command have not been supplied.", [Cmd]);
 
 format_error(Reason) ->
-    rebar_api:debug("Unknown error : ~ts", [Reason]),
+    rebar_api:debug("Unknown error : ~ps", [Reason]),
```

Comment:
Would throw when the error is e.g. a tuple. (it was not `dialyzer`, but testing on an invalid repo.)